src/components/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  query: string;
+  onFormInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const Header: React.FC<Props> = ({ todos, query, onFormInputChange }) => {
+  const activeTodos = useMemo(() => {
+    return todos.filter(todo => !todo.completed);
+  }, [todos]);
```

Comment:
Seems that you have the same calculation in the footer. So, you can move it to the highest level and pass the value as a prop